source/client.c

```diff
@@ -907,6 +972,8 @@ static void s_subscribe_single_complete(
 
     struct subscribe_task_arg *task_arg = userdata;
 
+    AWS_LOGF_DEBUG(AWS_LS_MQTT_CLIENT, "Subscribe %d completed", packet_id);
```

Comment:
Log whether it was accepted or rejected
a rejected subscription might be worthy of a higher log level like WARNING?